src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1514,6 +1514,12 @@ void ApplicationManagerImpl::OnCertificateUpdateRequired() {
   GetPolicyHandler().OnPTExchangeNeeded();
 }
 
+bool ApplicationManagerImpl::GetPolicyCertificateData(std::string& data) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  data = GetPolicyHandler().RetrieveCertificate();
+  return true;
```

Comment:
@AKalinich-Luxoft Why do we need to return always `true` ?

Reply:
@AByzhynar because SecurityManager has several listeners and it have to iterate over all of them to get certificate data. Returning true from AM listener is a signal that some data was provided in output reference and SecurityManager should stop this loop and return the result. All other listeners returns false and in this case iteration loop will be continued.